### ISSUE TITLE
Misleading preference name KEY_LANGUAGE_VALUE = "languageDescription"

### ISSUE THREAD

**Issue by nicolas-raoul** [commons-app/apps-android-commons#4379]
Before the next release it would be great to fix this preference's naming:
```
public static final String KEY_LANGUAGE_VALUE = "languageDescription"
```
Both the variable name and the content should be rethought.

This preference stores **the user's preferred language when filling captions and descriptions**.
How about this?
```
public static final String DESCRIPTION_LANGUAGE = "descriptionLanguage"
```
I believe the `KEY_` part is not useful, am I mistaken?

If we release with that, it will be much harder to fix afterwards, I think.

At the same time, this should be made coherent:
```
private ListPreference langListPreference
```

**Comment by Ayan-10** [comment]
@nicolas-raoul I will fix it along with this #4366 PR.

**Comment by nicolas-raoul** [comment]
@Ayan-10 Thanks! :-)
